Content.Server/DeviceLinking/Systems/DeviceLinkSystem.cs

```diff
@@ -19,6 +19,31 @@ public override void Initialize()
         SubscribeLocalEvent<DeviceLinkSinkComponent, DeviceNetworkPacketEvent>(OnPacketReceived);
     }
 
+    public override void Update(float frameTime)
+    {
+        var query = EntityQueryEnumerator<DeviceLinkSinkComponent>();
+
+        while (query.MoveNext(out var uid, out var component))
+        {
+            if (component.InvokeLimit < 1)
+            {
+                component.InvokeCounter = 0;
+                continue;
+            }
+
+            if(component.InvokeCounter > 0)
+                component.InvokeCounter--;
+
+            if (component.InvokeCounter <= component.InvokeLimit)
+                continue;
+
+            component.InvokeCounter = 0;
+            var args = new DeviceLinkOverloadedEvent();
+            RaiseLocalEvent(uid, ref args);
+            RemoveAllFromSink(uid, component);
```

Comment:
It seems like this should just be moved to where the counter actually gets incremented. I know recursion due to network packets should happen over multiple ticks now, but there might always be some other system sometime later that somehow triggers repeated port invoking in a single tick, so it just feels safer to move it there?